docs/content/getting-started/perform-check.mdx

```diff
@@ -163,6 +164,49 @@ To check whether user `user:anne` has relationship `reader` with object `documen
 
 The result's `allowed` field will return `true` if the relationship exists and `false` if the relationship does not exist.
 
+### 03. Calling Batch Check API
+
+If you want to check multiple user-object-relationship combinations in a single request, you can use the [Batch Check](/api/service#Relationship%20Queries/BatchCheck) API endpoint. Batching authorization checks together in a single request significantly reduces overall network latency.    
+
+The Batch Check endpoint requires a `correlation_id` parameter for each check. The `correlation_id` is used to "correlate" the check respones with the checks sent in the request, since `tuple_keys` and `contextual_tuples` are not returned in the response on purpuse to reduce data transfer to improve network latency. A `correlation_id` can be composed of any string of alphanumeric characters or dashes between 1-36 characters in length. 
```

Comment:
nit: found a typo - `purpuse` -> `purpose`

Reply:
nice find. I have a spell checker in vs code, but it isn't working.